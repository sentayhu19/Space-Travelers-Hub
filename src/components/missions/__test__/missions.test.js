import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../../../redux/configureStore';
import Missions from '../missions';

const Mission = [
  {
    missionName: 'mission1',
    key: '1',
    id: '1',
    description: 'Falcon 1',
    IsJoined: false,
  },
];

describe('Test the mission Component', () => {
  it('mission section renders', () => {
    const { container } = render(
      <Provider store={store}>
        <Missions key={Mission[0].id} mission={Mission[0]} />
      </Provider>,
    );

    expect(container).toMatchSnapshot();
  });
});
