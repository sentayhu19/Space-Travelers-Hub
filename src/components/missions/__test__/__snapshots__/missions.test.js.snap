// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Rockets Component rockets section renders 1`] = `
<div>
  <div>
    <table>
      <thead>
        <tr>
          <td>
            Mission
          </td>
          <td>
            Description
          </td>
          <td>
            Status
          </td>
          <td />
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;

exports[`Test the mission Component mission section renders 1`] = `
<div>
  <div>
    <table>
      <thead>
        <tr>
          <td>
            Mission
          </td>
          <td>
            Description
          </td>
          <td>
            Status
          </td>
          <td />
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;

exports[`Test the mission Component rockets section renders 1`] = `
<div>
  <div>
    <table>
      <thead>
        <tr>
          <td>
            Mission
          </td>
          <td>
            Description
          </td>
          <td>
            Status
          </td>
          <td />
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
